installer/upgrader/Migrations/V5/leaveLangString.yaml

```
@@ -0,0 +1,291 @@
+    unitId: leave_list
+  -
+    value: 'Show Leave with Status'
+    unitId: show_leave_with_status
+  -
+    value: 'My Leave Request Details'
+    unitId: my_leave_request_details
+  -
+    value: 'Leave Request Details'
+    unitId: leave_request_details
+  -
+    value: 'My Leave List'
+    unitId: my_leave_list
+  -
+    value: 'Show Leave with Status'
+    unitId: show_leave_with_status
+  -
+    value: 'Updating Entitlement'
+    unitId: updating_entitlement
+  -
+    value: 'Matching Employees'
+    unitId: matching_employees
+  -
+    value: 'The selected leave entitlement will be applied to the following employees.'
+    unitId: selected_leave_entitlement_applied_to_following_employees
+  -
+    value: 'Old Entitlement'
+    unitId: old_entitlement
+  -
+    value: 'New Entitlement'
+    unitId: new_entitlement
+  -
+    value: 'Situational Leave'
+    unitId: situational_leave
+  -
+    value: "These leave will be excluded from reports unless there's some activity. E.g. maternity leave, jury duty leave."
+    unitId: entitlement_situational_description
+  -
+    value: 'Confirm Leave Assignment'
+    unitId: confirm_leave_assignment
+  -
+    value: 'Employee does not have sufficient leave balance for leave request.'
+    unitId: employee_does_not_have_enough_balance_for_leave_request
+  -
+    value: 'Click OK to confirm leave assignment.'
+    unitId: click_ok_to_confirm_leave_assignment
+  -
+    value: 'Existing Entitlement value {oldvalue} will be updated to {newvalue}'
+    unitId: entitlement_value_confirmation_message
+  -
+    value: 'Leave Balance'
+    unitId: leave_balance
+  -
+    value: 'Updating Entitlement'
+    unitId: updating_entitlement
+  -
+    value: 'Balance not sufficient'
+    unitId: balance_not_sufficient
+  -
+    value: 'Insufficient Leave Balance'
+    unitId: insufficient_leave_balance
+  -
+    value: 'Balance'
+    unitId: balance
+  -
+    value: 'Days'
+    unitId: days
+  -
+    value: 'Available Balance'
+    unitId: available_balance
+  -
+    value: 'Leave Balance Details'
+    unitId: leave_balance_details
+  -
+    value: 'As of Date'
+    unitId: as_of_date
+  -
+    value: 'Total Entitlement'
+    unitId: total_entitlement
+  -
+    value: 'Leave Status'
+    unitId: leave_status
+  -
+    value: 'Taken'
+    unitId: taken
+  -
+    value: 'Scheduled'
+    unitId: scheduled
+  -
+    value: 'Pending Approval'
+    unitId: pending_approval
+  -
+    value: '{action, select, APPROVE {{count, plural, =0 {You are about to Approve no Leave Requests} =1 {You are about to Approve 1 Leave Request} other {You are about to Approve # Leave Requests} }} REJECT {{count, plural, =0  {You are about to Reject no Leave Requests} =1 {You are about to Reject 1 Leave Request} other {You are about to Reject # Leave Requests} }} other {{count, plural, =0    {You are about to Cancel no Leave Requests} =1    {You are about to Cancel 1 Leave Request} other {You are about to Cancel # Leave Requests} }} }'
```

```suggestion
    value: '{action, select, APPROVE {{count, plural, =0 {You are about to Approve no Leave Requests} =1 {You are about to Approve 1 Leave Request} other {You are about to Approve # Leave Requests} }} REJECT {{count, plural, =0  {You are about to Reject no Leave Requests} =1 {You are about to Reject 1 Leave Request} other {You are about to Reject # Leave Requests} }} other {{count, plural, =0 {You are about to Cancel no Leave Requests} =1 {You are about to Cancel 1 Leave Request} other {You are about to Cancel # Leave Requests} }} }'
```
will remove unnecessary spaces

---

removed